IMC_R/EDA_Raymond/EDA-R.md

```diff
@@ -0,0 +1,123 @@
+
+## IMDb vs Rotten Tomatoes
+IMDb and Rotten Tomatoes are two of the most popular sites for movie ratings. Given that they both perform a similar function, it would be reasonable to expect that the score a movie receives on IMDb would be correlated to the score it receives on Rotten Tomatoes, despite the fact that their scoring systems are quite different (IMDb scores come from audiences, while Rotten Tomatoes scores come from critics).
+
+Before we begin, let’s take a look at the distribution of scores from each site.
+
+![](https://i.imgur.com/u7A7KRv.png)![](https://i.imgur.com/QgiMv0U.png)
+
+Right away, something seems very wrong. The histogram of IMDb scores looks great, but there seems to be far too many near 0 observations for Rotten Tomatoes scores. It’s likely that we are looking at missing values, so we’ll need to look through our data. It’s worth noting here that since Rotten Tomatoes rates movies with a percentage, R has identified this as a factor variable, instead of a numeric one. To get proper results, we needed to convert these values to numbers with `as.numeric()`.
+
+![](https://i.imgur.com/pUMPbdq.png)
+
+Just as suspected, many of the movies are missing Rotten Tomatoes scores. There are a number of ways we could handle this, one of which is to create a new dataframe of movies that are missing this rating, and to then remove these movies from our original dataframe.
+
+![](https://i.imgur.com/KctYyDx.png)
+
+Now, let’s look at our Rotten Tomatoes histogram again.
+
+![](https://i.imgur.com/fCcGusO.png)
+
+This is much better. Although it still isn’t nearly as pretty as the IMDb distribution, we should be able to use this to find the correlation between the two sites. Getting the actual r value for correlation is super easy in R, we just need to use the `cor()` function with two numerical lists. We can also use ggplot, a package from Tidyverse, to create a scatter plot so that we can visualize this relationship. In the `ggplot()` function call, tomatoes is our dataset of movies with non-empty Rotten Tomatoes scores, and we specify IMDb as the x-axis and Rotten Tomatoes as the y-axis. The `geom_point()` component at the end is used to create the actual scatterplot.
+
+![](https://i.imgur.com/6LUfDqw.png)
+
+Our correlation coefficient, 0.377, tells us that there is a moderate relationship between IMDb and Rotten Tomatoes scores, and we can see this in the scatterplot (as IMDb scores increase, Rotten Tomatoes scores tend to increase as well). However, the relationship is a lot weaker than you might’ve expected, which tells us that audiences often perceive movie quality fairly differently from the critics.
+
```

Comment:
you may be interested in the Pearson-Correlation test to support the claim that 0.377 is a "moderate relationship" just fyi